src/systems/filecoin_mining/sector/sealing.id

```diff
@@ -26,7 +27,7 @@ type SealVerifyInfo struct {
     SealCfg
     Randomness             SealRandomness
     InteractiveRandomness  InteractiveSealRandomness
-    PieceInfos             [PieceInfo]
+    UnsealedCID            UnsealedSectorCID  // CommD
```

Comment:
aren't we using the `PieceInfos` to verify the sector on chain? how are we going to make sure `UnsealedSectorCID` (`CommR`) relates to relevant `PieceCIDs` (`CommPs`)?

Reply:
This is exactly what is computed by `StorageMarketActorCode_I.GetUnsealedCIDForDealIDs` (which relies indirectly on `filproofs` to do so). We populate the `SealVerifyInfo` only with the correct `UnsealedCID`. If the `PieceInfos` themselves were invalid (say, too large for the sector), then we will fail earlier without an `UnsealedSectorCID`.

Reply:
got it